desktop/core/src/desktop/auth/api_authentications.py

```diff
@@ -49,10 +52,17 @@ def authenticate(self, request):
 
     LOG.debug('JwtAuthentication: got access token from %s: %s' % (request.path, access_token))
 
+    # Handle public key
+    public_key_pem = ''
+    if AUTH.JWT.VERIFY.get():
+      jwk = requests.get(AUTH.JWT.KEY_SERVER_URL.get())
+      public_key = jwt.algorithms.RSAAlgorithm.from_jwk(json.dumps(jwk["keys"][0])).public_key()
```

Comment:
This will 500 blow-up if we need proper auth or json is improper, but good enough for v1

Reply:
Maybe check `AUTH.JWT.KEY_SERVER_URL.get()` is not empty, maybe same if jwk.get('keys') is empty